definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -1693,6 +1693,13 @@
     "UserRole": "citizen",
     "CRUD": "CRU"
   },
+  {
+    "LiveFrom": "01/03/2019",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "testDNPronounced",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
```

Comment:
citizen

Reply:
cases are put into this state by caseworkers, in real life...

Reply:
I don't think you want the caseworker see this event on real life.
All the test events should be triggered by citizen as we have all the control on citizen access

Reply:
oh ok. fair enough. didn't know that.
I'll change it.

Reply:
The problem is that as it stands, the caseworker will see it in the dropdown and be able to trigger it

Reply:
fair enough. changed it.